api/project.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+
+unless Dir.exists? ".project/common"
+  unless system(*%W{mkdir -p .project})
+    STDERR.puts "mkdir failed."
+    exit 1
+  end
+  unless system(*%W{git clone https://github.com/dmohs/project-management.git .project/common})
```

Comment:
What do you think of having this as a git submodule, instead of an ignored auto-download?